common/src/cellular_at_core.c

```diff
@@ -760,11 +777,11 @@ CellularATError_t Cellular_ATStrtoi( const char * pStr,
     }
     else
     {
-        /* MISRA C 2012 Directive 4.7 � If a function returns error information, then
+        /* MISRA C 2012 Directive 4.7 � If a function returns error information, then
```

Comment:
is this intentional?

Reply:
No, I really don't know why it's modified, maybe it's caused by the editor.
Will roll back its modification.